test/fixtures/output/kafka/mirror/describe-mirror-json.golden

```diff
@@ -7,7 +7,19 @@
     "status_time_ms": 111111111,
     "partition": 0,
     "partition_mirror_lag": 142857,
-    "last_source_fetch_offset": 1293009
+    "last_source_fetch_offset": 1293009,
+    "error_message": "",
+    "error_code": "",
```

Comment:
Should we hide these if they're empty? You can annotate this field with `omitempty` if so

Reply:
I'm confused why they are showing up now. These are existing fields are have always been:
```
	ErrorMessage                string                     `human:"Error Message" serialized:"error_message"`
	ErrorCode                   string                     `human:"Error Code" serialized:"error_code"`
```
See https://github.com/confluentinc/cli/blob/main/internal/kafka/command_mirror.go#L20


I can try using `omitempty` nonetheless.

Reply:
Actually, is adding `omitempty` not a breaking change given those fields already exist? I'm wondering if they originally were not in the output because they were the last fields in the struct but now they're not?

Reply:
Maybe they weren't included in the filter? Since they weren't displayed before, it should be OK to mark them `omitempty`!

Reply:
Good point. Makes sense. Added `omitempty`.